test/replication/long_row_timeout.result

```diff
@@ -54,15 +54,17 @@ test_run:cmd('setopt delimiter ";"')
 ---
 - true
 ...
-ok = true;
+status = nil;
 ---
 ...
 start = fiber.time();
 ---
 ...
+replication_timeout = box.cfg.replication_timeout
+box.cfg{replication_timeout = replication_timeout * 5}
```

Comment:
This test checks that the replica doesn't timeout too fast, so you have to change the timeout on the replica as well. Otherwise you're just making the test wait too long, and that's why you have 'status ==  stopped' below.

If you do that, the test will become much longer, because now it ends right when replica times out, which's pretty fast, because replica's timeout is 0.1 seconds AFAIR.

Reply:
Right, all corrected, thanks.